src/weights.jl

```diff
@@ -559,19 +556,32 @@ function wsum(A::AbstractArray{<:Number}, w::UnitWeights, dim::Int)
     return sum(A, dims=dim)
 end
 
-# extended sum! and wsum
+## extended sum! and wsum
 
 Base.sum!(R::AbstractArray, A::AbstractArray, w::AbstractWeights{<:Real}, dim::Int; init::Bool=true) =
-    wsum!(R, A, values(w), dim; init=init)
+    wsum!(R, A, w.values, dim; init=init)
```

Comment:
`values` isn't guaranteed to exist for any weights type. Anyway, we don't need it:
```suggestion
    wsum!(R, A, w, dim; init=init)
```